packages/paste-core/components/combobox/__tests__/index.spec.tsx

```diff
@@ -277,6 +317,22 @@ describe('Combobox', () => {
       // @ts-ignore Property 'value' does not exist on type 'HTMLElement' (I get it, but this is right)
       expect(screen.getByRole('textbox').value).toEqual('Alert');
     });
+
+    it('should not run react virtual\'s "scroll to" function when "groupItemsBy" is defined as a string', () => {
+      // bc grouped comboboxes are not yet virtualized
+      render(<GroupedMockCombobox />, {wrapper: ThemeWrapper});
+
+      const targetIndex = 1;
+      const target = groupedItems[targetIndex];
+
+      userEvent.hover(screen.getByText(target.label));
```

Comment:
Hover to trigger state change `highlightedIndex`, which controls the effect.

Reply:
Oh, this answers one of my questions above - so even if the list doesnt actually scroll it still triggers the state change?

Reply:
Yeah the highlighted index is coupled to mouseover position, not scroll position.